frontend/src/views/applications/SelectFacilityView.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <v-form ref="form" v-model="isFormComplete" class="mx-4">
+    <h1>Begin an Operating Funding Model application</h1>
+    <div class="mt-8">
+      <h4>Organization information</h4>
+      <div>
+        <v-icon>mdi-information-slab-circle-outline</v-icon>
+        Please review the following pre-populated information for correctness and contact your organization's account manager to make updates if required.
+      </div>
+      <OrganizationInfo />
+      <v-checkbox :value="1" :rules="rules.required" label="I confirm that Organization information is correct."></v-checkbox>
+    </div>
+    <div class="mb-6">
+      <h4>To start your application, select a facility</h4>
+      <div>
+        <v-icon>mdi-information-slab-circle-outline</v-icon>
+        If your facility is not listed, contact your Account Manager.
+      </div>
+      <v-row no-gutters class="mt-4">
+        <v-col cols="12" md="4" lg="2">
+          <div>Select your facility:</div>
+        </v-col>
+        <v-col cols="12" md="8" lg="10">
+          <v-select
+            id="facility"
+            v-model="model.facilityId"
+            :items="facilities"
+            item-title="facilityName"
+            item-value="facilityId"
+            :rules="rules.required"
+            chips
```

Comment:
I think we've only been using chips for multi select. I think the default style looks better for single select.

Reply:
it is removed